front/plugin.form.php

```diff
@@ -46,23 +46,26 @@
 $plugin = new Plugin();
 
 $id     = isset($_POST['id']) && is_numeric($_POST['id']) ? (int) $_POST['id'] : null;
-$action = $id > 0 && isset($_POST['action']) ? $_POST['action'] : null;
+$action = $_POST['action'] ?? null;
 
 switch ($action) {
     case 'install':
-        $plugin->install($id);
-        break;
     case 'activate':
-        $plugin->activate($id);
-        break;
     case 'unactivate':
-        $plugin->unactivate($id);
-        break;
     case 'uninstall':
-        $plugin->uninstall($id);
-        break;
     case 'clean':
-        $plugin->clean($id);
```

Comment:
Is it normal that this code is deleted ? It seems related to actions out of scope from this PR (does this mean plugins can no longer be installed from the UI ?).

Reply:
The actions will be executed by `$plugin->{$action}($id);`. The goal was to mutualize the `if (!$id) { ... }` block.